doc/neps/roadmap.rst

```diff
@@ -68,69 +70,119 @@ documentation on many topics are missing or outdated. See :ref:`NEP44` for
 planned improvements. Adding more tutorials is underway in the
 `numpy-tutorials repo <https://github.com/numpy/numpy-tutorials>`__.
 
-Our website (https://numpy.org) was completely redesigned recently. We aim to
-further improve it by adding translations, more case studies and other
-high-level content, and more (see `this tracking issue <https://github.com/numpy/numpy.org/issues/266>`__).
+We also intend to make all the example code in our documentation interactive -
+work is underway to do so via ``jupyterlite-sphinx`` and Pyodide.
+
+Our website (https://numpy.org) is in good shape. Further work on expanding the
+number of languages that the website is translated in is desirable. As are
+improvements to the interactive notebook widget, through JupyterLite.
 
 
 Extensibility
 -------------
 
-We aim to make it much easier to extend NumPy. The primary topic here is to
-improve the dtype system - see :ref:`NEP41` and related NEPs linked from it.
-Concrete goals for the dtype system rewrite are:
-
-- Easier custom dtypes:
+We aim to continue making it easier to extend NumPy. The primary topic here is to
+improve the dtype system - see for example :ref:`NEP41` and related NEPs linked
+from it. In NumPy 2.0, a `new C API for user-defined dtypes <https://numpy.org/devdocs/reference/c-api/array.html#custom-data-types>`__
+was made public. We aim
+to encourage its usage and improve this API further.
```

Comment:
Maybe add ", including support for writing a dtype in Python."? I think that part of the old list is still relevant (and would allow experimenting with dtype metadata perhaps).

Reply:
Good point, that progressed for a bit and then I'm not sure what happened to it. Edit sounds good to me if it's still in someone's plan to do this. @seberg can you comment?